### ISSUE TITLE
Module not found: error when using <Link>

### ISSUE THREAD

**Issue by assapir** [primer/components#801]
code example:
```
import Link from "@primer/components/src/Link";
import React from "react";

interface ExternalLinkProps {
  src: string;
  text: string;
  newTab?: boolean;
}

export const ExternalLink = (props: ExternalLinkProps) => {
  const target: string = props.newTab ? `_blank` : `_self`;
  const rel: string | undefined = props.newTab
    ? `noreferrer noopener`
    : undefined;

  return (
    <Link href={props.src} target={target} rel={rel}/>
  );
};
```
While trying to run `npm start` in console, it won't render, but show the following error:
`Module not found: Can't resolve './utils/elementType' in '{my path}/node_modules/@primer/components/src'`

**Comment by emplums** [comment]
Hey @assapir thanks for the 🐛 report! We're aware of this issue and shipping a fix in `19.0.0` which should go out any day now :) Here's the tracking PR if you'd like to follow along: #764

**Comment by assapir** [comment]
Thanks for that. The problem is that in mean while it's completely unusable :(

**Comment by dmarcey** [comment]
Can you do:
`import {Link} from "@primer/components";`
as a work-around?

**Comment by assapir** [comment]
yes, that works. but the bundle size is much bigger then. 
Waiting for 19.0.0!

**Comment by BinaryMuse** [comment]
This is now released in `v19.0.0`. See detailed instructions at https://primer.style/components/getting-started

**Comment by assapir** [comment]
Thannks for the follow up!
